docs/articles/index.html

```diff
@@ -95,6 +95,28 @@
         </div>
         <div class="feature">
             <span class="name">
+                Smart Assertions
+            </span>
+            <div class="description">
+                <p>TestCafe provides a full-featured set of <a href="documentation/test-api/assertions/assertion-api.html">built-in assertions</a>, so you do not need to reference additional libraries. Everything you need at  your fingertips. The built-in assertions work fine with the <a href="documentation/test-api/selecting-page-elements/selectors.html#snapshot-api-shorthands">Selector Shorthands</a> and <a href="documentation/test-api/selecting-page-elements/selectors.html#dom-node-snapshots">Snapshots</a>. All of this allows you to use the <a href="documentation/test-api/assertions/index.html#smart-assertion-query-mechanism">Assertion Query Mechanism</a> and create stable, fast and reliable tests, that do not depend on page response speed</p>
```

Comment:
Instead of `The built-in assertions work fine with...` maybe rephrase it to `When used with Selector`s DOM node state properties assertions enable Smart Assertion Query Mechanism..`.